File: BackEnd/backend/src/main/java/com/fitapp/backend/auth/aplication/dto/response/RegisterResponse.java
package com.fitapp.backend.auth.aplication.dto.response;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class RegisterResponse {
    @JsonProperty("email")
    private String email;

    @JsonProperty("message")
    private String message;

    @JsonProperty("requiresEmailVerification")
    private boolean requiresEmailVerification;
}

File: FrontEnd/app/src/main/java/com/fitapp/appfit/feature/auth/model/RegisterResponse.kt
package com.fitapp.appfit.feature.auth.model

import com.google.gson.annotations.SerializedName

data class RegisterResponse(
    @SerializedName("email") val email: String,
    @SerializedName("message") val message: String,
    @SerializedName("requiresEmailVerification") val requiresEmailVerification: Boolean = true
)
